src/usng.ts

```diff
@@ -1339,12 +1339,12 @@ extend(Converter.prototype, {
       return 0;
     }
 
-    strregexp = new RegExp("^[0-9]{2}[CDEFGHJKLMNPQRSTUVWX]$");
+    strregexp = new RegExp("^([1-9]|[1-5][0-9]|60)[CDEFGHJKLMNPQRSTUVWX]$");
```

Comment:
It looks like in usng4j, the coordinate part of the string is optional. Which implementation do we want? https://github.com/codice/usng4j/pull/11/files#diff-a692ac7744aec78853858fe4f9ad7592R48